features/e2e_tests/e2e_reactjs_app_tests.feature

```diff
@@ -0,0 +1,60 @@
+Feature: E2E Successfull payments on reactjs app
+
+  As a user
+  I want to use card payments method embeded on reactjs pages
+  In order to check full payment functionality
+
+  Background:
+    Given User opens reactjs app page with payment form
+
+  @base_config
+  Scenario: React app - successfully processed payments with tabs change
+    And User fills payment form with defined card MASTERCARD_CARD
+    And User clicks Pay button
+    And Notification message with Payment has been successfully processed text is displayed
+    When User switch tab to 'Personal Data' in reactjs app
+    And User switch tab to 'Home' in reactjs app
+    And User fills payment form with defined card VISA_CARD
+    And User clicks Pay button
+    Then Notification message with Payment has been successfully processed text is displayed
+
+  @e2e_config_defer_init_true
+  Scenario: React app - successfully processed payments with tabs change and and deferinit config
+    And User fills payment form with defined card VISA_CARD
+    And User clicks Pay button
+    And Notification message with Payment has been successfully processed text is displayed
+    When User switch tab to 'Personal Data' in reactjs app
+    And User switch tab to 'Payment' in reactjs app
+    And User fills payment form with defined card MASTERCARD_CARD
+    And User clicks Pay button
+    Then Notification message with Payment has been successfully processed text is displayed
+
+  @e2e_config_update_jwt_true
+  Scenario: React app - successfully processed payments with tabs change and update JWT
```

Comment:
We don't have 'update_jwt_true' property in config. 
UpdateJwt function is triggered by filling amount field and we have dedicated step for this:
And User calls updateJWT function by filling amount field

Reply:
changes will be added with next commit within next task (this branch i deleted after merge)  - thanks for clarification